lib/uktt/export_chapter_pdf.rb

```diff
@@ -488,15 +490,15 @@ def update_prs(v2_commodity)
       if @prs[id]
         @prs[id][:commodities] << v2_commodity.data.attributes.goods_nomenclature_item_id
       else
-        desc = v2_commodity.included.select{|obj| obj.type = "measure_type" && obj.id == id}.first.attributes.description
+        desc = v2_commodity.included.select { |obj| obj.type = 'measure_type' && obj.id == id }.first.attributes.description
         conditions_ids = measure.relationships.measure_conditions.data.map(&:id)
-        conditions = v2_commodity.included.select{|obj| obj.type = "measure_condition" && conditions_ids.include?(obj.id) }
+        conditions = v2_commodity.included.select { |obj| obj.type = 'measure_condition' && conditions_ids.include?(obj.id) }
         conditions.each do |condition|
-          unless condition.nil?
-            doc_code = condition.attributes.document_code
-            document_codes << condition.attributes.document_code 
-            requirements << "#{condition.attributes.condition_code}: #{strip_tags(condition.attributes.requirement)}#{" (#{doc_code})" unless doc_code.to_s.empty?}" unless condition.attributes.requirement.nil?
-          end
+          next if condition.nil?
```

Comment:
Are you sure this is correct? Because according to them the condition should be `if !condition.nil?` here.